### ISSUE TITLE
Create a management structure for Gear Store

### ISSUE THREAD

**Issue by edrushka** [MozillaFoundation/plan#423]
Driver: @edrushka 
Consulted:  @valianttry Jennifer Balaco from MoCo

Context: The Gear Store creates revenue for the Foundation: http://note.io/1akdW1J. Currently Geoffrey owns it. As he transitions, Andrea can take temporary ownership, but it needs a long-term management structure. A governance module has been proposed as a potential structure, with members from MoCo, MoFo and the community.

Flagging @davidascher to take this to Ops for Mark and Angela's input

**Comment by davidascher** [comment]
Cool send me said proposed governance model?

**Comment by valianttry** [comment]
@davidascher I am a complete n00b to modules. I'm on the policy module list, but don't know how to create one, what that entails; etc. I see this in the wiki: https://wiki.mozilla.org/Modules.

What Geoffrey and I have discussed (not in stone, just broad brushing):
* Mozilla Foundation receives about 40% net profit from everything sold through gear.mozilla.org. We've led thus far for 1.5 yrs on contracts, setup, and management so far.
* We have 2 vendors that do 90% of the work of the store (customer support, site maintenance and updates, sample production, fulfillment/shipping; etc)
* The other 10% of the work is 
1) Making decisions about what merchandise to develop and sell based on community demand and needs; 
2) devising cool marketing campaign ideas and working collaboratively on a marketing calendar; 
3) creating designs for sales and other gear promotions for Mozilla-owned channels; 
4) working collaboratively with moco design to make sure we all love the quality and maintain alignment with our overall brand guidelines; 
5) listening to community demands and ideas to incorporate into merchandising and marketing. 

A module can help mostly with #s 1, 2 and 5 above (with guidance from a core group of leaders). Three and four are foundation responsibilities (unless we decide to farm them out to a firm). Moco has helped with design support for their owned channels (mozilla.org homepage + FF+Y newsletter; etc).

Jennifer Balaco runs Moco's / Staples "internal/contributor gear" operations. She's been an ongoing consultant for us and we've coordinated closely with her as we've stood up our public store. Before John Slater left, we'd floated the idea of potentially asking if Jennifer's time could be shared with us so she could help with managing the public-facing store as well. John's left and this was never really discussed (Matej is her new supervisor).

We'd originally estimated potential net revenue at $250 - 500K/yr for gear.mozilla.org. With proper marketing that seems entirely possible -- as long as we invest in 1 - 5 above.

**Comment by edrushka** [comment]
adding @RebsD. Welcome to Github, Rebecca!

**Comment by davidascher** [comment]
This probably needs a synchronous conversation, but I want to flag the difference between a management structure and a governance model.  Modules are about the latter, not the former.

From a management point of view, we need someone to drive action; the module structure can be useful to encourage participation, make sure that the work is aligned with more stakeholders than just said management structure, etc.  We don't need a module to allow participation, by the way -- lots of parts of Mozilla have room for volunteer contribution without a clear module frame.

I would say the module system is particularly helpful if it helps resolve conflicts, and enshrines a review system which can leverage domain expertise and long term accountability, even as management roles shift over the years.

Regardless, it's pointless to talk about a module system if there aren't specific leaders who are passionate about the work and willing to commit to building that module up, including making the time necessary to make the module successful.  Module ownership is built around module owners, who have proven excellence and deeply care about the topic at hand.

Can discussions about what should be in the store be public?  What are the current topics under discussion w.r.t. the gear store?  Or is the problem that nobody is driving, and as a result things are stalling?

**Comment by edrushka** [comment]
Moving to next heartbeat. This has stalled a bit.

**Comment by OpenMatt** [comment]
status?

**Comment by edrushka** [comment]
No progress. Fundraising team is getting their 2015 priorities in place, with @RebsD's leadership.

**Comment by RebsD** [comment]
A cost-benefit analysis with suggested scenarios for Gear Store management will be presented to Ops this Friday, May 29.

**Comment by edrushka** [comment]
@valianttry and I met with Jennifer Balaco from MoCo in Whistler. We're awaiting details of her contract to see if/how she can assist with management.

The "module" governance model is no longer being discussed at the moment. We need someone to run the Gear Store first.

@valianttry can you follow up with Jennifer and report back?

**Comment by RebsD** [comment]
I've updated Andrea on the following, sorry not to have extended it to
Erika and git hub:

In Europe, I met with Jascha and Mary Ellen. They agree the gear store
situation is sub-optimal, recognize there's potential, and are committed to
working with us to see how it could intersect with
brand/marketing/participation.

Next step: Andrea to review her original recommendation document and
circulate to MEM and Jascha and me.

RD

**Comment by OpenMatt** [comment]
@RebsD @valianttry status?

**Comment by RebsD** [comment]
In conversation with Mary Ellen Mukerman about shared or transferred
responsibility.

**Comment by OpenMatt** [comment]
@RebsD : update / move to a future milestone as needed?

**Comment by RebsD** [comment]
yes

**Comment by edrushka** [comment]
Update: @RebsD and @valianttry met with MoCo in San Francisco to discuss this. If either of you have more details you can share from that meeting, please add.

**Comment by valianttry** [comment]
@RebsD and I met with Mary Ellen Muckerman and Alex Costello. I created a document detailing the tasks and duties involved in managing gear.mozilla.org and we walked through it in detail with them:
https://docs.google.com/document/d/1B5Qz-Mqc0P2zjEmCo8a53A9gvtKClsnqePd4yKnbpVk/edit

Next steps:
We all agreed the store has big potential, and is a worthy endeavor for our brand and community. Existing relationships with vendors are going well. We are set up for success but the store needs more investment in marketing and management to really grow.

Jennifer Balaco was just hired full time to help project manage on the Brand team (yay!). One of her responsibilities is managing the internal Staples store, and she may pick up some responsibilities for the public store (gear.mozilla.org). Alex Costello is going to talk to Jennifer about her capacity and streamlining processes.

**Comment by edrushka** [comment]
@valianttry Any updates from Alex or Jennifer?

**Comment by valianttry** [comment]
Email exchange with Alex Costello this week pasted below.
TLDR: No help soon from Moco. Given my core fundraising responsibilities, I do not have capacity to continue managing gear.mozilla.org. 
TASK: @RebsD to present case to hire Gear Store management support at Ops meeting on 8/25

Email from Alex Costello 8/24: ------------------
There is so much going on!
Yes, Jen Balaco is working on an analysis for the gear store; her time, Staples v ThinkGeek work and overall process. This audit isn't going to be finalized for a week or so. I don't think we are anywhere near scoping and staffing discussions at this time - and if that even falls into this team is still a question as well. 

Let's maybe regroup after Labor Day? When is this $100 gift cert going live?

Thanks

On Mon, Aug 24, 2015 at 10:20 AM, Andrea Wood <andrea@mozillafoundation.org> wrote:

    Hi Alex,

    I just had a conversation with Mardi Douglas. Apparently Chris B wants to provide $100 coupon codes to every employee for the Firefox Birthday. That functionality is being built into the public gear store right now, but I'm not really able to give it the urgency it needs because of all my other commitments.

    After our last discussion, I believe next steps was for Jennifer to evaluate her processes / time commitments. Can we circle up soon on the scoping and staffing for gear store management?

    Thanks!

    andrea

**Comment by RebsD** [comment]
I've asked for budget to get support managing gear store in the meantime.

**Comment by OpenMatt** [comment]
@rebsd: I'll roll this over to the next Sep 11 heartbeat -- unless that's wrong.

**Comment by OpenMatt** [comment]
@edrushka status?

**Comment by RebsD** [comment]
A request has been made from Andrea to Angela to hire someone to manage
gear store in the interim (while discussions continue with MoCo about a
transfer or partnership)

**Comment by hannahkane** [comment]
@edrushka - is this ticket still needed?

**Comment by edrushka** [comment]
I'll leave that for @valianttry and @davidascher to answer.

**Comment by hannahkane** [comment]
@valianttry @davidascher - can we close or move to a more accurate milestone?

**Comment by hannahkane** [comment]
I'm moving this to a future milestone, so we can close this one.

**Comment by edrushka** [comment]
I'm going to close this.
